ffi/derive/tests/ui_pass/valid.rs

```diff
@@ -1,21 +1,21 @@
-use std::{collections::BTreeMap, mem::MaybeUninit};
+use std::collections::BTreeMap;
```

Comment:
I think the filename can be improved. `valid` gives nothing new to me, because I already know that it should be valid by looking at `ui_pass` folder name

Reply:
it holds a variety of test all of which should compile. I learnt recently that passing compile tests actually get executed so I might move them to integration tests

Reply:
> I learnt recently that passing compile tests actually get executed so I might move them to integration tests

Good idea, since they'll be considered examples by downstream users.